grails-app/conf/application.yml

```diff
@@ -211,6 +211,69 @@ lists:
     facets: true
     migratoryDR: 'dr1005'
     iconicSpeciesDR: 'dr781'
+    version: 1
+    listTypes:
```

Comment:
We probably should export this from lists, similar to how logger does it for reasons codes.

Reply:
1 == specieslist-webapp
2 == species-lists

Any suggestions?